debbuild.spec

```diff
@@ -57,6 +57,7 @@ Recommends:     quilt
 Recommends:     unzip
 Recommends:     zip
 Recommends:     zstd
+Recommends:     gem
```

Comment:
This has a distro-varying dependency, so it needs to go up in the vendor conditional block above.

`rubygems` for RPM distributions and `ruby` for Debian distributions.

Reply:
Done. I think I need to drink some water because `gem` isn't even the package I thought it was. :expressionless: